types/font-carrier/.eslintrc.json

```diff
@@ -0,0 +1,5 @@
+{
+    "rules": {
+        "dt-header": "off"
```

Comment:
Please don't disable this or any other lint rule, for the header specifically, we use metadata from the header for automation, thanks~

Reply:
ok